docker/Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV NODE_ENV=production
 WORKDIR /src
 
 COPY . .
+RUN npm install -g npm@latest
```

Comment:
Did out of date npm cause problems in this ticket? Could update the base image if you want to use a more recent npm. Oh, is this related to rollup issue you describe below? Either way could maybe put a comment here because I'm a still a bit confused!

Reply:
I don't know if it was related, but it was something I did while trying to debug the rollup issue. While working on that, I updated my local npm version, which then put my machine 1 minor version ahead of the one used in the Docker container. It seemed like a good idea to use the latest npm available in both places?